spec/index.html

```diff
@@ -31,54 +32,57 @@ <h1>Ecmarkup</h1>
-  <emu-table id="build-options">
+  <p>Build options and document options can be passed in via the command line or specified in the <emu-xref href="#metadata">front-matter</emu-xref> of your document.</p>
+  <emu-table id="build-options" class="code">
     <emu-caption>Build options</emu-caption>
     <table>
-      <tr><th>Option</th><th>Description</th></tr>
-      <tr><td>watch</td><td>Rebuild when files change</td></tr>
-      <tr><td>verbose</td><td>Print verbose logging info</td></tr>
-      <tr><td>write-biblio</td><td>Emit a biblio file to the specified location</td></tr>
-      <tr><td>assets</td><td>"none" for no css/js, "inline" for inline css/js</td></tr>
-      <tr><td>css-out</td><td>Emit the Ecmarkup CSS file to the specified location</td></tr>
-      <tr><td>js-out</td><td>Emit the Ecmarkup JS file to the specified location</td></tr>
-      <tr><td>lint-spec</td><td>Enforce some style and correctness checks</td></tr>
-      <tr><td>lint-formatter</td><td>The <a href="https://eslint.org/docs/user-guide/formatters/">eslint formatter</a> to be used for printing warnings and errors when using --verbose. Either the name of a built-in eslint formatter or the package name of an installed eslint compatible formatter.</td></tr>
-      <tr><td>strict</td><td>Exit with an error if there are warnings. Cannot be used with --watch</td></tr>
-      <tr><td>mark-effects</td><td>Whether to render <a href="#effects">effects</a> like "user code".</td></tr>
+      <tr><th>Command Line Option</th><th>Front-Matter Key</th><th>Description</th></tr>
+      <tr><td>`--watch`</td><td></td><td>Rebuild when files change.</td></tr>
+      <tr><td>`--verbose`</td><td></td><td>Print verbose logging info.</td></tr>
+      <tr><td>`--write-biblio`</td><td></td><td>Emit a biblio file to the specified path.</td></tr>
+      <tr><td>`--assets`</td><td>`assets`</td><td>"none" for no CSS/JS, "inline" for inline CSS/JS.</td></tr>
+      <tr><td>`--css-out`</td><td>`cssOut`</td><td>Emit the Ecmarkup CSS file to the specified path.</td></tr>
+      <tr><td>`--js-out`</td><td>`jsOut`</td><td>Emit the Ecmarkup JS file to the specified path.</td></tr>
+      <tr><td>`--lint-spec`</td><td>`lintSpec`</td><td>Enforce some style and correctness checks.</td></tr>
+      <tr><td>`--error-formatter`</td><td></td><td>The <a href="https://eslint.org/docs/user-guide/formatters/">eslint formatter</a> to be used for printing warnings and errors when using `--verbose`. Either the name of a built-in eslint formatter or the package name of an installed eslint compatible formatter.</td></tr>
+      <tr><td>`--strict`</td><td></td><td>Exit with an error if there are warnings. Cannot be used with `--watch`.</td></tr>
+      <tr><td>`--mark-effects`</td><td>`markEffects`</td><td>Propagate and render <a href="#effects">effects</a> like "user code".</td></tr>
+      <tr><td>`--multipage`</td><td></td><td>Emit a distinct page for each top-level clause.</td></tr>
```

Comment:
I'm not clear on the difference between "Build options" vs. "Document options", but it does seem strange for `markEffects` to not be in the same category as `toc`.

Reply:
I'm not entirely clear on the difference either, but I agree `markEffects` probably ought to be under "document options".